lib/ansible/modules/network/nxos/nxos_vrf.py

```diff
@@ -379,10 +379,12 @@ def map_params_to_obj(module):
 
 
 def get_value(arg, config, module):
```

Comment:
What is the point of `arg` here, and how does it differ from `arg` below?

Reply:
`arg` takes the value of the parameters (extra_params) in a loop in the `map_config_to_obj` method which calls the `get_value` method. Here, we check if arg is in the config as a key. Then we take its corresponding value.

Reply:
Oh my goodness, I misread the `if` below as `for`. Repeatedly.

Ignore me.